fmt/src/formatter.rs

```diff
@@ -1212,68 +1270,30 @@ mod tests {
         );
     }
 
-    #[test]
-    fn constructor_definition() {
-        test_directory("ConstructorDefinition");
-    }
-
-    #[test]
-    fn contract_definition() {
-        test_directory("ContractDefinition");
-    }
-
-    #[test]
-    fn error_definition() {
-        test_directory("ErrorDefinition");
-    }
-
-    #[test]
-    fn enum_definition() {
-        test_directory("EnumDefinition");
-    }
-
-    #[test]
-    fn event_definition() {
-        test_directory("EventDefinition");
-    }
-
-    #[test]
-    fn function_definition() {
-        test_directory("FunctionDefinition");
-    }
-
-    #[test]
-    fn import_directive() {
-        test_directory("ImportDirective");
-    }
-
-    #[test]
-    fn modifier_definition() {
-        test_directory("ModifierDefinition");
-    }
-
-    #[test]
-    fn statement_block() {
-        test_directory("StatementBlock");
-    }
-
-    #[test]
-    fn struct_definition() {
-        test_directory("StructDefinition");
-    }
-
-    #[test]
-    fn type_definition() {
-        test_directory("TypeDefinition");
-    }
-
-    #[test]
-    fn using_directive() {
-        test_directory("UsingDirective");
+    macro_rules! test_directory {
+        ($dir:ident) => {
+            #[allow(non_snake_case)]
+            #[test]
+            fn $dir() {
+                test_directory(stringify!($dir));
+            }
+        };
     }
 
-    #[test]
-    fn variable_definition() {
-        test_directory("VariableDefinition");
-    }
+    test_directory! { ConstructorDefinition }
+    test_directory! { ContractDefinition }
+    test_directory! { DocComments }
+    test_directory! { EnumDefinition }
+    test_directory! { ErrorDefinition }
+    test_directory! { EventDefinition }
+    // test_directory! { ExpressionPrecedence }
```

Comment:
should we re-enable this?

Reply:
this was an experiment related to the operator precedence as mentioned above. i'll also remove